src/Catty/PlayerEngine/DataModel/CBSpriteNode/CBSpriteNode.swift

```diff
@@ -34,6 +34,7 @@ class CBSpriteNode: SKSpriteNode {
     }
     @objc var currentUIImageLook: UIImage?
 
+    var rotationMode: RotationStyle
```

Comment:
Please rename to `rotationStyle` (instead of mode) to have the same naming convention everywhere.

Reply:
Done. ;D